### PR TITLE
Signed-off-by: Pritam Saha <saha7pritam@gmail.com>

### PR DESCRIPTION
**Pull request by Saha-7**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by freben — CHANGES_REQUESTED** [review]

File: plugins/newrelic/src/api/index.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ type Options = {
 };
 
 export interface NewRelicApi {
-  getApplications(): Promise<NewRelicApplications>;
+  getApplications(apiKey: string): Promise<NewRelicApplications>;
```
[the remark this answers is not in the document]
Comment (Saha-7) [reply plugins/newrelic/src/api/index.ts:71]: Okay, I will try to complete this. If any issue happened  I will let you know.

**Comment by Saha-7** [comment]
@freben I raised a  fresh PR for this in #17677 .
You can continue there & close this PR. 
Please let me know if other changes required .

**Comment by Saha-7** [comment]
@freben is the PR still open or closed?

**Comment by freben** [comment]
Well it's open right? The other one was closed in favor of this one, and this one still does not have a green status.

**Pull request closed without merging**